include/rocksdb/env.h

```diff
@@ -94,6 +94,12 @@ struct EnvOptions {
   // See DBOPtions doc
   size_t random_access_max_buffer_size;
 
+  // See DBOptions doc
+  size_t writable_file_max_buffer_size = 1024 * 1024;
+
+  // See DBOptions doc
+  bool skip_table_builder_flush = false;
```

Comment:
This is doesn't not belong to EnvOptions. It's arguable should be CFOptions or BlockBasedTableOptions, but Env should not have any knowledge of "table".